public/index.html

```diff
@@ -32,8 +32,13 @@ <h1>JASON JOHN</h1>
         <a id="openModalButton" href="https://withjoy.com/jason-and-esha/rsvp" target="_blank">RSVP HERE</a>
 
     </div>
+    <!-- Info Button -->
+    <button class="info-button" id="infoButton">
+        <img src="info.png" alt="Info">
+    </button>
     <div id="info_page">
         <div class="sidenav">
+            <br> 
```

Comment:
This adds a bit of FORCED space between the sidebar items and the info button image, on a few weird aspect ratio browsers/devices.